<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemSettingPage</class>
 <widget class="QWidget" name="SystemSettingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(83, 83, 83);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="settingTab">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="workflowTab">
      <attribute name="title">
       <string>Workflow</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="imageCommentTab">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <attribute name="title">
       <string>Image Comment</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="connectionTab">
      <attribute name="title">
       <string>Connection</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="collimatorTab">
      <attribute name="title">
       <string>Collimator</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="dapTab">
      <attribute name="title">
       <string>DAP</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="pacsEntityTab">
      <attribute name="title">
       <string>PACS Entity</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="worklistTab">
      <attribute name="title">
       <string>Worklist</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="procedureTab">
      <attribute name="title">
       <string>Procedure</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="viewTab">
      <attribute name="title">
       <string>View</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="techniqueSettingTab">
      <attribute name="title">
       <string>Technique Setting</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="imageProcessTab">
      <attribute name="title">
       <string>Image Process</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="generatorTab">
      <attribute name="title">
       <string>Generator</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="detectorTab">
      <attribute name="title">
       <string>Detector</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(208, 208, 208);</string>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
